lib/index.js

```diff
@@ -137,6 +137,12 @@ module.exports = mergeExports(fn, {
 	get DllReferencePlugin() {
 		return require("./DllReferencePlugin");
 	},
+	get DynamicEntryPlugin() {
+		return require("./DynamicEntryPlugin");
```

Comment:
Why do you need to expose this?

Reply:
It's not really needed for this PR, but missing in general. `EntryPlugin` is exposed, so should be `DynamicEntryPlugin`, which is the counterpart for function entries.